docs/src/pages/components/data-grid/virtualization/virtualization.md

```diff
@@ -33,9 +33,13 @@ By default, 2 columns are rendered outside of the viewport. You can change this
 
 You can disable column virtualization by setting the column buffer to a higher number than the number of rendered columns, e.g. with `columnBuffer={columns.length}` or `columnBuffer={Number.MAX_SAFE_INTEGER}`.
 
-## apiRef [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
+## Disable virtualization
 
-{{"demo": "pages/components/data-grid/virtualization/VirtualizationApiNoSnap.js", "bg": "inline", "hideToolbar": true}}
```

Comment:
Why did you remove this section?

Reply:
One of the changes is to move the scroll methods to a separated hook. Since this interface is empty now I removed it. Maybe we should add a page for scrolling with a demo: https://github.com/mui-org/material-ui-x/issues/1103#issuecomment-885695797. In this new page we could document the API available.